src/go/k8s/controllers/redpanda/console_controller.go

```diff
@@ -452,7 +459,7 @@ func getDeploymentStrategy(console *redpandav1alpha1.Console) v1.DeploymentStrat
 func (r *ConsoleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&redpandav1alpha1.Console{}).
-		Owns(&redpandav1alpha1.Cluster{}).
```

Comment:
This is right change. I copied from cluster controller that will not work in this setup.